core/job/service/job_service.go

```diff
@@ -24,6 +24,15 @@ import (
 const (
 	ConcurrentTicketPerSec = 50
 	ConcurrentLimit        = 100
+
+	metricTotalJobAddSuccess = "total_job_add_success"
+	metricJobUpdateSuccess   = "job_update_success"
+	metricJobDeleteSuccess   = "job_delete_success"
+
+	metricTotalJobUpsertFailed = "total_jobs_upsert_failed"
+	metricTotalJobDeleteFailed = "total_jobs_delete_failed"
+
+	metricTotalUpstreamRefresh = "total_upstream_refresh"
```

Comment:
correct, it was not intended to, but this will be the same number as modified/added. initially, this metric should cover the number of **jobs** that the upstream has been successfully resolved.

we can remove this metric, and add `total_jobs_with_unresolved_upstream` or `total_unresolved_upstream` instead. this can be easily added in `getUnresolvedUpstreamsErrors` function. wdyt?

Reply:
had discussion for this, we decided to not publish metric for upstream refresh.